photon-server/src/main/java/org/photonvision/vision/pipeline/Calibrate3dPipeline.java

```diff
@@ -146,7 +146,7 @@ protected CVPipelineResult process(Frame frame, Calibration3dPipelineSettings se
                         MathUtils.nanosToMillis(sumPipeNanosElapsed),
                         fps,
                         Collections.emptyList(),
-                        new Frame(new CVMat(inputColorMat), frame.frameStaticProperties));
+                        new Frame(new CVMat(outputColorMat), frame.frameStaticProperties));
```

Comment:
Note - this is going to potentially conflict functionally with #185 . I'd changed this to assist with debugging in the unit sim and at runtime.

Depending on the merge order, some code may need to be tweaked. The underlying key - there are two `mat`'s in play here. Whichever is _not_ getting returned is the one that needs `release()` called on it.

May also be impacted by some of the discord discussion about architecture and using thresholded vs color frames from picam accelerated sources.

Reply:
Redid 185 so this should no longer be a concer.